openstates/tx/tests/test_vote_parsing.py

```diff
@@ -1,27 +1,31 @@
 import os
 
-from nose.tools import *  # noqa
+import unittest
```

Comment:
This switch from nose to unittest was to assuage the flake8 linter.

Also, I'm not sure how to run these. I think the [instructions on the readme](https://github.com/openstates/openstates/#testing) work for billy scrapers but not pupa, or I've broken something here.